docs/pages/includes/determinetls.mdx

```diff
@@ -0,0 +1,26 @@
+<Details title="How to tell if TLS Routing is Enabled" opened={false}>
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+To get details of the Teleport Cluster you can invoke `/webapi/ping` path
+on the proxy service.  The `proxy.tls_routing_enabled` value provides if TLS routing is enabled.
```

Comment:
```suggestion
on the Teleport Proxy Service. The `proxy.tls_routing_enabled` value indicates if TLS Routing is enabled.
```

Reply:
recreating in my edits since I'm making other changes to this line.